Client/Frontend/Theme/ThemeManager.swift

```diff
@@ -49,13 +57,23 @@ final class DefaultThemeManager: ThemeManager {
     // MARK: - Variables
 
     var currentTheme: Theme = LightTheme()
+    var notificationCenter: NotificationProtocol
     private var userDefaults: UserDefaults
+    private var appDelegate: UIApplicationDelegate?
 
     // MARK: - Init
 
-    private init(userDefaults: UserDefaults = UserDefaults.standard) {
-        self.userDefaults = userDefaults
-        self.currentTheme = generateTheme()
+    private init() {
+        self.userDefaults = AppContainer.shared.resolve()
+        self.notificationCenter = AppContainer.shared.resolve()
+        self.appDelegate = AppContainer.shared.resolve()
```

Comment:
Why do we want to resolve those from the container? Instead of example using `UserDefaults.standard` as a default param in the init